### PR TITLE
Tch4lla/hamburger menu

### LINKED ISSUE #55: Navbar - Wire up the hamburger menu logic
# Requirements

- Make hamburger menu clickable
- The css should follow [tailwind css](http://tailwindcss.com/) best practices.
- Add a visual css indicator that it has been clicked
- Through vue best practices, this should trigger showing the hamburger menu and update the `state` in the navbar vue component.
- Note: the hamburger menu itself will be created in a separate PR.

This has a dependency on PR #54

### PR DESCRIPTION
**Pull request by Tch4lla**
Resolves #55 

# What changed

- changed the h1 menu items to spans and display block
- changed the options api format to composition api. Less boilerplate code and readability is easier 
- hamburger can be clicked and displays menu items 

# Testing instructions
-reduce screen width to 800px
-click the hamburger menu

### CODE REVIEW

File: components/TopNav.vue [hunk 1]
```diff
@@ -6,35 +6,36 @@
...
             </NuxtLink>
         </div>
-        <div>
+        <div >
             <NuxtLink to="/about">
                 About
             </NuxtLink>
-            <div class="cursor-pointer md:hidden">
-                <img src="../assets/images/hamburgerMenu.svg" alt="hamburger menu" />
+            <div  class="cursor-pointer md:hidden " >
```
Comment (ermish) [inline components/TopNav.vue:13]: looks like you need to run your linter. Try running `yarn lint --fix`

File: components/TopNav.vue [hunk 2]
```diff
@@ -6,35 +6,36 @@
...
+                <img @click="toggleMenu" src="../assets/images/hamburgerMenu.svg" alt="hamburger menu" />
+                <div v-if="showMenu">
```
Comment (ermish) [inline components/TopNav.vue:15]: for this PR, this approach is fine. You should consider how you plan to show these same links in within the hamburger menu in the future work.

**Review by ermish — CHANGES_REQUESTED** [review]
Thanks @Tch4lla for the contribution! I added a few comments.

File: components/TopNav.vue (hunk 1 above)
Comment (Tch4lla) [reply components/TopNav.vue]: Ran linter and fixed issue with spacing

**Review by ermish — APPROVED** [review]
LGTM! :shipit:

**Pull request merged**